src/AccessibilityInsights.SharedUx/Resources/Styles.xaml

```diff
@@ -1563,4 +1563,8 @@
             </Trigger>
         </Style.Triggers>
     </Style>
+    <Style TargetType="{x:Type RadioButton}" x:Key="primaryFGRadioButtonStyle" BasedOn="{StaticResource {x:Type RadioButton}}">
```

Comment:
There may be opportunities to leverage this style to replace explicitly setting the FG color in XAML on a per-control basis, but I'll look at that as a separate PR once this is merged.